backend/internal/handlers/classrooms/assignments/works/works.go

```diff
@@ -281,6 +282,39 @@ func (s *WorkService) GetCommitCount() fiber.Handler {
 	}
 }
 
+func (s *WorkService) GetCommitsPerDay() fiber.Handler {
+    return func(c *fiber.Ctx) error {
+        work, err := s.getWork(c)
+		if err != nil {
+			return err
+		}
+
+
+        
+        // Unsure how this will work for group assignments
```

Comment:
this is a case where you could easily just total the commits for each contributor